### PR TITLE
Add iOS 15.6 support, also link to 15.5 in the readme

### PR DESCRIPTION
**Pull request by clintron**
Note that I pulled these files from the Xcode 14 beta 5.  I've imported them into Xcode 13.3 and have been able to debug an iOS 15.6.1 device.

### CODE REVIEW

**Review by atulkhatri — APPROVED** [review]

**Comment by clintron** [comment]
@atulkhatri I'm not able to merge this PR; would you mind doing that, or are you waiting for other maintainers to review?

**Comment by atulkhatri** [comment]
> @atulkhatri I'm not able to merge this PR; would you mind doing that, or are you waiting for other maintainers to review?

Sorry I don't have access to merge  it.

**Comment by filsv** [comment]
@clintron Thanks, added.

**Pull request merged**
